gen/python/sym/util.py

```diff
@@ -8,14 +8,17 @@
 
```

Comment:
`typing` is not a builtin package in python 2.  We could add it as a depenency of `sym` or something, but that seems like a bummer.  I'm not actually sure what a good approach would be here - maybe @eric-skydio has ideas?  For context we have this mildly-complicated type that we'd like an alias for, but ideally we don't add a runtime dependency on `typing` from this module (`sym`)

Noting that the import is actually added [here](https://github.com/symforce-org/symforce/pull/251/commits/c307b52d7ef457c71fcd50dd7c372fd9ce0e1b32#diff-d25fb3617e7ff49546425e19e9e5d6338a20b0f831ab60f32c2cb5235ae2a242R7), although it isn't necessary at runtime in that PR.